.travis.yml

```diff
@@ -6,13 +6,15 @@ python: "2.7"
 env:
```

Comment:
I saw that you're extending the python support in the ``tox.ini`` file, but you're not executing the new test suite here. Fill the ``TOX_ENV`` list.